package.json

```diff
@@ -151,9 +150,11 @@
     "danger": "^10.6.0",
     "enzyme": "^3.11.0",
     "enzyme-to-json": "^3.6.1",
-    "eslint": "^7.12.1",
-    "eslint-config-amex": "^11.2.0",
+    "eslint": "^7.32.0",
+    "eslint-config-amex": "^14.2.1",
```

Comment:
Why aren't we going to 15?

Reply:
15 has another 200+ changes, most of which require a significant effort